ONSdigital/ons-boxen
# Create dhilton manifest

**What_**

This PR creates the manifest for user dhilton. This is an intial
setup for creating a developer's mac laptop at the ONS. A separate PR will follow explaining how to create a new profile in the repo and how to bootstrap a new mac using boxen.

**How to test_**
1. Acquire a new mac
2. Follow the boxen quickstart notes in README.md 
3. Run Boxen as dhilton (using github authorisation)
4. Check that the expected applications are installed (Virtualbox, DockerToolBelt, etc)

**Who can test**

Probably only @dhilton
